addons/mail/models/discuss/discuss_channel.py

```diff
@@ -1197,7 +1202,8 @@ def _create_channel(self, name, group_id):
         new_channel = self.create(vals)
         group = self.env['res.groups'].search([('id', '=', group_id)]) if group_id else None
         new_channel.group_public_id = group.id if group else None
-        notification = Markup('<div class="o_mail_notification">%s</div>') % _("created this channel.")
+        current_partner, __ = self.env["res.partner"]._get_current_persona()
+        notification = Markup('<div class="o_mail_notification">%s</div>') % _("%(user)s created this channel.", user=current_partner.name)
```

Comment:
What if a guest created the channel? We usually don't ignore the 2nd current persona result.